### ISSUE TITLE
UnusedPrivateElementsSniff should consider __clone

### ISSUE THREAD

**Issue by marcospassos** [slevomat/coding-standard#550]
This sniff currently reports an unused `__clone` method:

```php
class Foo {
    // Disable object cloning
    private function __clone(){}
}
```

I want to suggest not warning for this case in particular, as is the standard way of making an object not cloneable.

**Comment by kukulich** [comment]
Thank you for your report.

Fixed in https://github.com/slevomat/coding-standard/commit/796e342a114d38b0aa5823f19417d0965072395a
